lib/stat_tracker.rb

```diff
@@ -222,23 +222,28 @@ def team_goals_league_total
   end
 
   def avg_team_goals_league
-    team_games_league_total.reduce(Hash.new) do |team_avgs, (team, tot_games)|
-      team_goals_league_total.each do |team, tot_goals|
-        team_avgs[team] = (tot_goals.to_f / tot_games).round(2)
+      team_avgs = Hash.new
+      @teams.each do |team|
+        team_games = team_games_league_total[team.team_id]
```

Comment:
are team_games and team_goals essentially representing the same data?

Reply:
nah, one is goals throughout the season, the other is a counter of total games played